### PR TITLE
ISAICP-5281: Fixed maps on events.

### PR DESCRIPTION
**Pull request by junkuncz**

### CODE REVIEW

**Comment by pfrenssen** [comment]
It seems like the use case that was originally reported is not covered by a test.

Let's add a test that adds a physical location, checks that the map is displayed, then edits the event, removes the phyical location, and then checks that the map is no longer displayed.

In order to do this we will need to propose a PR for the https://github.com/openeuropa/oe_webtools_location project to add the step definition for `Then I should not see a map on the page` in `OpenEuropaWebtoolsMapsSubContext`.

File: tests/features/joinup_event/event.edit.feature [hunk 1]
```diff
@@ -4,6 +4,24 @@ Feature: "Event page" editing.
   As an owner of the event
   I need to be able to edit it.
 
+  Scenario: Add and remove map
+    Given collections:
+      | title            | logo     | banner     | state     |
+      | Stream of Dreams | logo.png | banner.jpg | validated |
```
Comment (pfrenssen) [inline tests/features/joinup_event/event.edit.feature]: This is using the same name of a collection that is already used in another test. Please use a unique name so we can easily recognize failing tests.

File: tests/features/joinup_event/event.edit.feature [hunk 2]
```diff
@@ -4,6 +4,24 @@ Feature: "Event page" editing.
...
+    And I am logged in as a facilitator of the "Stream of Dreams" collection
+    When I go to the homepage of the "Stream of Dreams" collection
+    And I click "Add event" in the plus button menu
+    When I fill in the following:
+      | Title             | An amazing event                      |
```
Comment (pfrenssen) [inline tests/features/joinup_event/event.edit.feature]: There is already an event used with this name, description and location in the `add_event.collection` test. Please choose a unique name, description and location.

File: tests/features/joinup_event/event.edit.feature (hunk 1 above)
Comment (junkuncz) [reply tests/features/joinup_event/event.edit.feature]: Changes added.

File: tests/features/joinup_event/event.edit.feature (hunk 2 above)
Comment (junkuncz) [reply tests/features/joinup_event/event.edit.feature]: Changes added.

**Pull request merged**